example/token/app.go

```diff
@@ -48,16 +49,32 @@ func (we WebAdapter) tokenAuthWrapFunc(handler http.HandlerFunc, permissions []s
 			return
 		}
 
-		err = we.tokenAuth.ValidatePermissionsClaim(claims, permissions)
+		err = we.tokenAuth.ValidateScopeClaim(claims, scope)
```

Comment:
For this example we should use `AuthorizeRequestScope` instead of `ValidateScopeClaim`. If we do this, we can remove the `permissions` and `scope` parameter from `tokenAuthWrapFunc`. Since these are now handled internally by Casbin, they should not be passed in.